digits/config/caffe_option.py

```diff
@@ -187,6 +189,9 @@ def get_version(executable):
         elif platform.system() == 'Darwin':
             # XXX: guess and let the user figure out errors later
             return (0,11,0)
+        elif platform.system() == 'Windows':
+            # XXX: guess and let the user figure out errors later
+            return (0,12,0)
```

Comment:
Can we guess `v0.11` for Windows too?

I'm working on a patch for Caffe that exposes the version number as `caffe.__version__`, but until then I think we should just assume the worst.